.github/workflows/makefile.yml

```diff
@@ -0,0 +1,53 @@
+# On push events to the master branch, this workflow will build, test, and create a release
+# of the fgdc2iso WAR file.
+
+name: 'Test and (Maybe) Release'
+
+on:
+  push:
+    branches:
+    - '**'
+
+jobs:
+  make:
+    name: 'Make'
+    runs-on: ubuntu-latest
+
+    # Use the Bash shell regardless whether the GitHub Actions runner is ubuntu-latest, macos-latest, or windows-latest
+    defaults:
+      run:
+        shell: bash
+
+    # Checkout the repository to the GitHub Actions runner
+    steps:
+    - uses: actions/checkout@v2
+      with: 
+        fetch-depth: '0'
+
+    - name: Copy saxon license to the license file
+      run: 'echo "$VAULT_FGDC2ISO_SAXON_LICENSE" > saxon-license.lic'
+      env: 
+        VAULT_FGDC2ISO_SAXON_LICENSE: ${{ secrets.VAULT_FGDC2ISO_SAXON_LICENSE }}
+
+    - name: Build and run tests
+      run: make all
+
+    - name: Bump version and push tag
+      id: bump
+      if: github.ref == 'refs/heads/master' && github.event_name == 'push'
```

Comment:
Instead of repeating the conditional for any publish action, is there a way to split the tag + release into it's own job or workflow that consumes the war file as an output? That would make this a little cleaner and less error prone (in case someone forgets adding the `if` to a new publish task).